.buildkite/pipeline.yaml

```diff
@@ -2,6 +2,7 @@ env:
   KAIZEN_ROLE_ARN: ${BRANCH_ROLE_ARN}
   KAIZEN_DOMAIN_NAME: ${BRANCH_DOMAIN_NAME}
   KAIZEN_DISTRIBUTION_ID: ${BRANCH_DISTRIBUTION_ID}
+  GITHUB_REGISTRY_TOKEN: ${GITHUB_REGISTRY_TOKEN}
```

Comment:
TODO: If we're not using theme-switcher, we might not need these anymore (though we might want them in future?)

Reply:
Agree(ish). The way we're heading we may include private packages in this repo in the future. I'll rip this out of this PR as we don't need it here anymore.